keras/utils/generic_utils.py

```diff
@@ -5,15 +5,27 @@
 import six
 import marshal
 import types as python_types
+from functools import partial
 
 
 def get_from_module(identifier, module_params, module_name,
                     instantiate=False, kwargs=None):
     if isinstance(identifier, six.string_types):
-        res = module_params.get(identifier)
+        idx = identifier.find('(')
+        if -1 == idx:
+            res = module_params.get(identifier)
+        else:
+            func_name = identifier[:idx]
+            aux_kwargs = eval('dict' + identifier[idx:])
```

Comment:
What happens if I pass in a harmful string like `"(injection=__import__('os').listdir('.'))"`? Perhaps the identifier string should just be parsed for k,v pairs, using `set` or `setattr`.